packages/css-library/src/stylesheets/base/va.scss

```diff
@@ -386,7 +386,7 @@ article > h1 {
     padding: scale-rule(1rem 0);
   }
 
-  background: url('#{$image-path}/stars.png') no-repeat center;
+  background: url(~@department-of-veterans-affairs/css-library/dist/img/stars.png) no-repeat center;
```

Comment:
ah nice, cool to see that this works. I wasn't sure if we would need to rewrite relative paths or not. Hopefully we can just get away with referencing the node nodules path this way :crossed_fingers: